apps/backend/src/users/users.controller.ts

```diff
@@ -17,7 +18,7 @@ export class UsersController {
 
   @Get()
   @ApiOperation({ summary: 'Get all users' })
-  async findAll(): Promise<User[]> {
```

Comment:
it might be worth creating a new DTO, like UserWithoutWeight, that is the same just without the weight field. Or the standard User entity could stay as well, it's not like we would use the weight field on the frontend